pkg/orchestrator/etcd_worker.go

```diff
@@ -257,6 +254,13 @@ func (worker *EtcdWorker) Run(ctx context.Context, session *concurrency.Session,
 	}
 }
 
+func isRetryableError(err error) bool {
+	err = errors.Cause(err)
+	return cerrors.ErrEtcdTryAgain.Equal(err) ||
+		cerrors.ErrReachMaxTry.Equal(err) ||
+		context.DeadlineExceeded == err
```

Comment:
I'm not sure if the `cerrors.ErrReachMaxTry and context.DeadlineExceeded `should handle here.